src/octoprint/settings.py

```diff
@@ -300,6 +300,7 @@ def settings(init=False, basedir=None, configfile=None):
 			"okWithLinenumber": False,
 			"numExtruders": 1,
 			"includeCurrentToolInTemps": True,
+			"includeFilenameOnOpened": True,
```

Comment:
This typo got fixed in the next commit.  Looks like I squashed it with the wrong one.